src/pages/HomePage/HomePage.tsx

```diff
@@ -199,16 +198,15 @@ function HomePage() {
               </Grid>
             </form>
             {
-              !flightSearchStatus.isLoading && flightSearchStatus.result?.error &&
+              !flightSearchStatus.isLoading && flightSearchStatus.result?.error && 
                 <Alert severity="error">
                   {flightSearchStatus.result!.error!.message}
                 </Alert>
             }
           </Grid>
 
```

Comment:
@jgarciadelgado-dxc kind of weird... this line was overwritten on the last merge... the merge didn't showed any conflict... but here is where I'm populating the passengers field.